admin/data/sql/migrations/20190815151201.undo.pupilgroup-vewpupilswithactivefamiliarisationpins.sql

```diff
@@ -0,0 +1,25 @@
+ALTER VIEW [mtc_admin].[vewPupilsWithActiveFamiliarisationPins] AS
+  SELECT
+         p.id,
+         p.foreName,
+         p.lastName,
+         p.middleNames,
+         p.dateOfBirth,
+         p.urlSlug,
+         p.school_id,
+         pin.val as pin,
+         cp.pinExpiresAt,
+         g.group_id
+  FROM [mtc_admin].[pupil] p
+         INNER JOIN [mtc_admin].[school] s ON (p.school_id = s.id)
+         LEFT JOIN  [mtc_admin].[pupilGroup] g ON (g.pupil_id = p.id)
```

Comment:
Why do we need to join to [group] here?

Reply:
thats the undo query, which does in fact need the group table.